### ISSUE TITLE
Discourse - Input fields in the Registreren modal don't have their instructions announced to screenreader users

### ISSUE THREAD

**Issue by savitris** [frameless/developer.overheid.nl#132]
**Dit probleem kan op meerdere pagina's aanwezig zijn, waaronder:**
<a href="https://community.developer.overheid.nl/t/officiele-bekendmakingen-binnenlandse-zaken-en-koninkrijksrelaties/726">Officiële bekendmakingen (Binnenlandse Zaken en Koninkrijksrelaties)</a>

**Probleembeschrijving:**
Probleem: De invoervelden in het modal dat opent met de knop "Registreren" tonen extra instructies, bijvoorbeeld "minimaal 10 tekens", die niet hardop worden voorgelezen door de screenreader wanneer gefocust op het gerelateerde invoerveld. Invoervereisten moeten programmatisch gekoppeld zijn aan hun velden.

Mogelijke verbeteringen: Hoewel de invoervelden al `aria-describedby` attributen hebben, staat de tekst van de extra instructies in een apart span-element dat niet verbonden is met het ID van het invoerveld. Werk het `aria-describedby` attribuut van elk invoerveld bij zodat het verwijst naar het span-element dat de instructietekst bevat. Dit kan gedaan worden door een ID toe te voegen aan de span die de extra instructies bevat (bijvoorbeeld: "minimaal 10 tekens"), en door dat ID op te nemen in het `aria-describedby` attribuut van het invoerveld.

Voorbeeldverbeteringencode:
```
<input autocomplete="current-password" placeholder="" id="new-account-password" class="ember-text-field ember-view" aria-describedby="password-validation more-info" aria-invalid="" type="password">

<span id="more-info" class="more-info">minimaal 10 tekens</span>
```

**Referentiecode:**
```
<div class="input-group create-account__password">
                  <input autocomplete="current-password" placeholder="" id="new-account-password" class="ember-text-field ember-view" aria-describedby="password-validation" aria-invalid="" type="password">
                  <label class="alt-placeholder" for="new-account-password">
                    Wachtwoord<span class="required">*</span>
                  </label>
                  <div class="create-account__password-info">
                    <div class="create-account__password-tip-validation">
                      <div id="password-validation" class="tip bad ember-view"><!----></div>
                      <span class="more-info">minimaal 10 tekens</span>
                      <div class="caps-lock-warning
                          hidden">
                        <svg class="fa d-icon d-icon-exclamation-triangle svg-icon svg-string" xmlns="http://www.w3.org/2000/svg"><use href="#exclamation-triangle"></use></svg>
                        Caps Lock is ingeschakeld
                      </div>
                    </div>
                    
    <button class="btn-link toggle-password-mask btn btn-text" title="Wachtwoord weergeven" type="button">
<!----><span class="d-button-label">Weergeven<!----></span>
    </button>
  
                  </div>

                <div class="password-confirmation">
                  <label for="new-account-password-confirmation">Wachtwoord opnieuw</label>
                  <input autocomplete="new-password" placeholder="" id="new-account-confirmation" class="ember-text-field ember-view" type="text">
                  <input id="new-account-challenge" class="ember-text-field ember-view" type="text">
                </div>
              </div>
```

**Gerelateerde WCAG:**
WCAG 3.3.2 Labels or instructions
Level A

**Referentieafbeelding:**

<img width="635" alt="A screenreader reads the placeholder of an input field but does not read the extra text instructions visible right under the input field" src="https://github.com/user-attachments/assets/9d7f859e-9b2c-4a44-894a-8b0e774d1b58" />

_**Referentienummer:** A128_
